src/main/kotlin/org/stellar/walletsdk/WalletSigner.kt

```diff
@@ -0,0 +1,9 @@
+package org.stellar.walletsdk
+
+import org.stellar.sdk.Transaction
+
+interface WalletSigner {
+  fun signWithClientAccount(txn: Transaction): Transaction
+
+  fun signWithDomainAccount(transactionString: String, networkPassPhrase: String): Transaction
```

Comment:
Ideally the function contract for both `signWithClientAccount()` and `signWithDomainAccount()` is the same `fun (transaction: Transaction): Transaction`